neqo-transport/src/connection/dump.rs

```diff
@@ -50,3 +55,26 @@ pub fn dump_packet(
         path.borrow()
     );
 }
+
+#[allow(clippy::too_many_arguments)]
+pub fn log_packet(
+    conn: &Connection,
+    path: &PathRef,
+    tos: IpTos,
+    dir: &Direction,
```

Comment:
An idea occurred as I was hitting "submit"...

It might be possible to move this direction to a marker trait.  Imagine if `QlogDirection` were a trait that was implemented by `Qlog{Send|Receive}Direction` with `fn qlog(...args) { qlog::packet_sent(...args) }` or `fn qlog(...args) { qlog::packet_received(...args) }` as necessary.  Then you could `use QLogSendDirection as Tx` and call this function as `log_packet::<Tx>(...args)`.  Just a thought.